tests/conformance/common.go

```diff
@@ -74,10 +81,27 @@ func LoadComponents(componentPath string) ([]v1alpha1.Component, error) {
 	return components, nil
 }
 
+func LookUpEnv(key string) string {
+	if val, ok := os.LookupEnv(key); ok {
+		return val
+	}
+
+	return ""
+}
+
 func ConvertMetadataToProperties(items []v1alpha1.MetadataItem) map[string]string {
 	properties := map[string]string{}
 	for _, c := range items {
-		properties[c.Name] = c.Value.String()
+		val := c.Value.String()
+		if strings.HasPrefix(c.Value.String(), "${{") {
+			// look up env var with that name. remove ${{}} and space
+			k := strings.TrimSpace(strings.TrimSuffix(strings.TrimPrefix(val, "${{"), "}}"))
+			v := LookUpEnv(k)
```

Comment:
Fail test/ print error if env var value is empty.

Reply:
Please don't fail parsing here using `log.Fatal`, since it will cause the other tests to fail as well.  Instead, can you please print a warning, or fail using `t.Error`?

Reply:
I cannot do a `t.Error` here, because if I did that it will not run the other tests in the configuration file. 
Instead I can continue to run other tests and pass an error back to the caller of `Run()` method which can then fail on behalf of this.